iCubGenova11/iKinGazeCtrl.ini

```
@@ -0,0 +1,11 @@
+robot icub
+head_version 2.7
```

```suggestion
head_version v2.7
```

@AntonioConsilvio @mfussi66, just commit the change through the button below.